src/test/groovy/edu/ie3/simona/model/participant/ChpModelTest.groovy

```diff
@@ -113,16 +112,16 @@ class ChpModelTest extends Specification {
     activePower.isEquivalentTo(getQuantity(expectedActivePower, KILOWATT))
 
     where:
-    chpState           | storageLvl | heatDemand  || expectedActivePower
-    chpStateNotRunning | 90         | 0           || 0                        // tests case (false, false, true)
-    chpStateNotRunning | 90         | 8 * 115     || 95                        // tests case (false, true, false)
-    chpStateNotRunning | 90         | 10          || 0                        // tests case (false, true, true)
-    chpStateRunning    | 90         | 0           || 95                        // tests case (true, false, true)
-    chpStateRunning    | 90         | 8 * 115     || 95                        // tests case (true, true, false)
-    chpStateRunning    | 90         | 10          || 95                        // tests case (true, true, true)
-    chpStateRunning    | 90         | 7 * 115 + 1 || 95                        // test case (_, true, false) and demand covered together with chp
-    chpStateRunning    | 90         | 9 * 115     || 95                        // test case (_, true, false) and demand not covered together with chp
-    chpStateRunning    | 92         | 1           || 95                        // test case (true, true, true) and storage volume exceeds maximum
```

Comment:
Are you sure we want to remove the double pipe separators? I remember them as the demarcation between test inputs (on the left side) and expected solutions (on the right side). I think they have no technical meaning, but they just signify the proposed usage of the variables to the developer.

Reply:
Technicaly no but I got annoyed by the codacy issue that is raised by this. So another way would be to adapt the codacy pattern, but I'm not sure if there can be some opt-out just for the groovy tests. In my point I would relinquish to have a demarcation line.

Reply:
Hmm well the double pipe is described for the spock testing framework [here](https://spockframework.org/spock/docs/2.3/data_driven_testing.html#_syntactic_variations). 
Codacy thinks the double pipe is a boolean operator here (which it could be in other contexts, but not here), so I think the error is with Codacy. I just disabled this particular check for Groovy files, if you're fine with it I'd just keep it like this, especially because Codacy in my view is just a supplementary code checker besides SonarQube.

Reply:
Fine for me, wasn't aware that we can filter these tests based on the type of the file.